package.json

```diff
@@ -42,7 +42,8 @@
     "start": "node dist/sth/bin/hub.js",
     "start:dev": "ts-node packages/sth/src/bin/hub.ts",
     "install:clean": "yarn clean && yarn clean:modules && yarn install",
-    "postinstall": "lerna link",
+    "install:python-deps": "pip install -r python/runner/requirements.txt --target python_modules",
```

Comment:
Hmm. I don't expect other python pakcages with dependencies, really - we just have python runner. So I would suggest to address this if an actual need arises, so that we don't overcomplicate things.

Reply:
I can pass this, but we will have packages in different languages (c++, rust) and I don't want to see copies of this. This is just to make sure that we use lerna.

Let's fix this in another pr.